Makefile

```diff
@@ -1,21 +1,25 @@
 SNOWBALL="./snowball/"
 ARABIC_STEMMER="algorithm/stemmer.sbl"
+ARABIC_BASED_ROOT_STEMMER = "algorithm/based_root_stemmer.sbl"
```

Comment:
ARABIC_ROOT_BASED_STEMMER , if you are using pycharm, use refactor -> rename feature